content/10-minute-tutorial.md

```diff
@@ -626,9 +629,10 @@ When("I ask whether is's Friday yet") do
   @actual_answer = is_it_friday?(@today)
 end
 
-Then("I should be told {string}") do |expected_answer|
+Then("I should be told {string}§:w") do |expected_answer|
```

Comment:
`§:w` is that slang for "I am leaving the vi editor"?

Reply:
@diabolo Would you be OK with me editing this, and then merging?

Reply:
Just go ahead @olleolleolle